boards/arm/mimxrt1060_evk/mimxrt1060_evk.dts

```diff
@@ -153,15 +154,19 @@ arduino_serial: &lpuart3 {
 	status = "okay";
 	width = <480>;
 	height = <272>;
-	hsync = <41>;
-	hfp = <4>;
-	hbp = <8>;
-	vsync = <10>;
-	vfp = <4>;
-	vbp = <2>;
-	polarity = <3>;
-	pixel-format = "bgr-565";
-	data-buswidth = "16-bit";
+	hsync-len = <41>;
```

Comment:
I think Linux uses a separate node for timing settings, see eg https://www.digi.com/resources/documentation/digidocs/90001945-13/reference/yocto/r_an_adding_custom_display.htm

Reply:
I've moved the timing settings into a child node. We can't exactly emulate linux here, because this snippet:

```
display-timings {
            native-mode = <&timing0>;
            timing0: hsd101pfw2 {
```

creates a circular reference, which Zephyr does not support. I instead implemented the devicetree like this:

```
display-timings {
            hsync-len <xx>;
            ...
```